src/components/dashboard/main-content/course-enrollments/course-cards/SavedForLaterCourseCard.jsx

```diff
@@ -0,0 +1,170 @@
+      {
+        key: 'move-course-to-in-progress',
+        type: 'button',
+        onClick: () => {
+          setIsModalOpen(true);
+          sendTrackEvent('edx.learner_portal.course.move_to_in_progress.modal.opened', {
+            course_run_id: courseRunId,
+          });
+        },
+        children: (
+          <div role="menuitem">
+            Move to In Progress
+            <span className="sr-only">for {title}</span>
+          </div>
+        ),
+      },
+    ]);
+  };
+
+  const renderButtons = () => {
+    if (isCourseEnded(endDate) || courseRunStatus === 'completed') { return null; }
+    return (
+      <ContinueLearningButton
+        linkToCourse={linkToCourse}
+        title={title}
+        courseRunId={courseRunId}
+      />
+    );
+  };
+
+  const renderCertificateInfo = () => (
+    props.linkToCertificate ? (
+      <div className="d-flex mb-3">
+        <div className="mr-3">
+          <img src={CertificateImg} alt="verified certificate preview" />
+        </div>
+        <div className="d-flex align-items-center">
+          <p className="lead mb-0 font-weight-normal">
+            View your certificate on{' '}
+            <a
+              className="text-underline"
+              href={`${process.env.LMS_BASE_URL}/u/${username}`}
+            >
+              your profile →
+            </a>
+          </p>
+        </div>
+      </div>
+    ) : (
+      <p className="lead mb-3 font-weight-normal">
+        To earn a certificate,{' '}
+        <a className="text-underline" href={props.linkToCourse}>
+          retake this course →
+        </a>
+      </p>
+    )
+  );
+
+  return (
+    <BaseCourseCard
+      buttons={renderButtons()}
+      dropdownMenuItems={getDropdownMenuItems()}
+      type="completed"
+      hasViewCertificateLink={false}
+      {...props}
+    >
+      {renderCertificateInfo()}
+      <MoveToInProgressModal
+        isOpen={isModalOpen}
+        courseTitle={title}
+        courseLink={linkToCourse}
+        courseId={courseRunId}
+        onClose={handleMoveToInProgressOnClose}
+        onSuccess={handleMoveToInProgressOnSuccess}
+      />
+    </BaseCourseCard>
+  );
+};
+
+SavedForLaterCourseCard.propTypes = {
+  linkToCourse: PropTypes.string.isRequired,
+  courseRunId: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  linkToCertificate: PropTypes.string,
+  savedForLater: PropTypes.bool.isRequired,
+  courseRunStatus: PropTypes.string.isRequired,
+  modifyCourseRunStatus: PropTypes.func.isRequired,
+  modifyIsMoveToInProgressCourseStatus: PropTypes.func.isRequired,
+  endDate: PropTypes.string,
+};
+
+SavedForLaterCourseCard.defaultProps = {
+  linkToCertificate: null,
+  endDate: null,
+};
+
+const mapDispatchToProps = dispatch => ({
+  modifyCourseRunStatus: (options) => {
+    dispatch(updateCourseRunStatus({ ...options }));
+  },
+  modifyIsMoveToInProgressCourseStatus: (options) => {
+    dispatch(updateIsMoveToInProgressCourseSuccess(options));
+  },
```

Comment:
this may be an existing issue but I just noticed, one of them we pass the spread options, but the other one just 'options', if not part of your work feel free to ignore it but just making a note of the difference here ... would be better to use the same convention (inputs as is vs with spread ...)

Reply:
oh, interesting. I don't _think_ I made that change, but I agree that having them consistent would be best.

Reply:
ok np, let's address it later then thanks for checking ... just did not want to lose track, so maybe you can leave a TODO note for now?

Reply:
I think it'll be just as easy to make the change now. :)

Reply:
(Although all this shows up as new code, it's just copied from the old CompletedCourseCard)

Reply:
OH! In looking into this I realized we no longer need these functions at all in the CompletedCourseCard. 🎉